server/neptune/workflows/internal/deploy/terraform/runner.go

```diff
@@ -31,6 +33,11 @@ func (r *WorkflowRunner) Run(ctx workflow.Context, deploymentInfo DeploymentInfo
 	id := deploymentInfo.ID
 	ctx = workflow.WithChildOptions(ctx, workflow.ChildWorkflowOptions{
 		WorkflowID: id.String(),
+		RetryPolicy: &temporal.RetryPolicy{
+			InitialInterval:    1 * time.Second,
+			BackoffCoefficient: 2.0,
+			MaximumInterval:    100 * time.Second,
+		},
```

Comment:
Lets hold off on this actually there's still some open questions I have and Im not convinced this is the best solution yet.